packit/distgit.py

```diff
@@ -353,7 +354,34 @@ def upstream_archive_names(self) -> list[str]:
                 if (s.remote or s.number == self.package_config.spec_source_id_number)
                 and s.valid
             ]
-        logger.debug(f"Upstream archive names: {archive_names}")
+        logger.debug(
+            f"Remote sources or sources specified by spec_source_id (Source0 by default): "
+            f"{archive_names}",
+        )
+        return archive_names
+
+    @property
+    def local_archive_names(self) -> list[str]:
+        """
+        Sources that are considered local (are not specified by URL).
+        Source specified by spec_source_id is excluded.
```

Comment:
Maybe I am missing something really obvious, but I probably do not fully understand why we are handling the `spec_source_id` sources separately. Can you please describe this a bit (at least for me ..;) or via comment)?

Reply:
I have extended the docstring to mention this.

Reply:
Thanks!